.expeditor/config.yml

```diff
@@ -1,5 +1,5 @@
 # Documentation available at https://expeditor.chef.io/docs/getting-started/
-
+---
```

Comment:
Since we're updating the config.yml, should we add a project alias to make promotions easier?